app/models/concerns/provideable.rb

```diff
@@ -12,13 +12,14 @@ module Provideable
       where(organization: organization).order(:business_name)
     }
 
-    def self.import_csv(csv, organization)
+    def self.import_csv(csv, organization, errors = [])
```

Comment:
Sorry, I'm a bit confused here. Why are we passing in an error array and then appending to it? Wouldn't the method always create an empty array and return it? You could make this pretty succinct by just having the last line of each method (where we aren't including any errors) as simply

```
[]
```

Reply:
I decided to assign the empty array to a variable `errors` to add a bit of clarity as to why we're returning an empty array.
I imagine that most `import_csv` methods (or at least donation site, per CL's comment) will eventually have some error checking like the one I'm implementing as part of the PR.

I can definitely go ahead with the suggested changes though if that's the style preferred here.